Source/Core/Common/ChunkFile.h

```diff
@@ -200,6 +200,16 @@ class PointerWrap
     DoArray(arr, static_cast<u32>(N));
   }
 
+  // The caller is required to inspect the mode of this PointerWrap
+  // and deal with the pointer returned from this function themself.
```

Comment:
```suggestion
  // The caller is required to inspect the mode of this PointerWrap
  // and deal with the pointer returned from this function itself.
```

Reply:
I suppose I'm humanizing the caller here as a "person who is writing the function calling this code". Is there any style guide for comments in general? I'm not really opposed to this but wouldn't be able to get around to making this change and squashing it until possibly tomorrow.

Reply:
That would only make sense if "The Caller" was a proper noun, in which case it must be capitalized like so and between parenthesis to make it explicit, but since it's not, then it should be "itself"

Reply:
Both ways seem awkward to me.  But this:

> The caller is required to inspect the mode of this PointerWrap and deal with the returned pointer by itself.

sounds a lot better than this:

> The caller is required to inspect the mode of this PointerWrap and deal with the returned pointer by themself.

Reply:
> 
> 
> ...what? I don't understand why assuming the caller is a human implies that it needs to be a proper noun. If you replace this with something like 'The doctor is required...' the word 'doctor' doesn't need to be capitalized either.
> 
> If you ask me the comment is fine as-is anyway, but I'm really not following your train of thought here.

Because "themself" is intended for people, and people have proper nouns.

Reply:
People can be referred to using nouns that are not proper nouns. And capitalizing a noun doesn't necessarily make it a proper noun either.